static-site/FAQ.md

```diff
@@ -36,46 +34,61 @@ permalink: /faq.html
   - [Does the BFD FHIR API provide information on benefit details such as deductibles, copays, coinsurance, etc.?](#does-the-bfd-fhir-api-provide-information-on-benefit-details-such-as-deductibles-copays-coinsurance-etc)
   - [How does United States Core Data for Interoperability (USCDI) relate to FHIR and its implementation within the BFD API?](#how-does-united-states-core-data-for-interoperability-uscdi-relate-to-fhir-and-its-implementation-within-the-bfd-api)
-
+  - [What other additional resources that are valuable to review in learning about BFD and its ecosystem?](#what-other-additional-resources-that-are-valuable-to-review-in-learning-about-bfd-and-its-ecosystem)
 <br/>
 
 ### BFD Basics
 
 #### What data is available via BFD?
 
-BFD provides different types of data on/about persons who are eligible for Medicare via various [FHIR](http://hl7.org/fhir) resources:
-- Patient resources that contain demographic data such as name, address, etc.
-- Coverage resources that contain enrollment data such as Part A status, Part D status, etc.
-- ExplanationOfBenefit resources that contain Part A, B, and D Medicare claims data, such as date of service, diagnosis codes, etc.
+
+FHIR, or Fast Healthcare Interoperability Resources, is a standard for exchanging healthcare information electronically. Developed by HL7 International, FHIR aims to simplify and streamline the interoperability of health data across different systems.
+
+Key features of FHIR include:
+- Resource-based Structure: FHIR organizes healthcare data into "resources," which represent discrete clinical or administrative concepts, such as patients, medications, and lab results.
+- RESTful APIs: It utilizes modern web technologies and principles, including RESTful APIs, allowing for easy integration and access to health information.
+- Interoperability: FHIR promotes interoperability between disparate health systems, enabling them to communicate and share data effectively.
+- Flexibility: It is designed to be adaptable, allowing developers to implement it in various environments and applications.
+
+Overall, FHIR aims to improve the accessibility, usability, and integration of healthcare information to enhance patient care.
 
 #### What does BFD stand for?
 
 Beneficiary FHIR Data Server
 
+#### What data is available via BFD?
+
+BFD provides different types of data on/about persons who are eligible for Medicare via various [FHIR](http://hl7.org/fhir) resources:
+- Patient resources that contain demographic data, such as name, address, etc.
+- Coverage resources that contain enrollment data, such as Part A status, Part D status, etc.
+- ExplanationOfBenefit resources that contain Part A, B, and D Medicare claims data, such as date of service, diagnosis codes, etc. An EOB shows the status of a claim and how it was processed. The EOB includes the amounts paid by a plan and amounts one is responsible to pay, if any. 
```

Comment:
Maybe it would be helpful to link Patient, Coverage, and ExplanationOfBenefit to their FHIR pages?

```suggestion
- [Patient](https://www.hl7.org/fhir/r4/patient.html) resources that contain demographic data, such as name, address, etc.
- [Coverage](https://www.hl7.org/fhir/r4/coverage.html) resources that contain enrollment data, such as Part A status, Part D status, etc.
- [ExplanationOfBenefit](https://www.hl7.org/fhir/r4/explanationofbenefit.html) resources that contain Part A, B, and D Medicare claims data, such as date of service, diagnosis codes, etc. An EOB shows the status of a claim and how it was processed. The EOB includes the amounts paid by a plan and amounts one is responsible to pay, if any. 
```